site/profile/manifests/base.pp

```diff
@@ -106,6 +106,19 @@
   file { glob('/tmp/terraform_*.sh'):
     ensure => absent,
   }
+
+  file_line { 'Include sshd_config.d':
+    path    => '/etc/ssh/sshd_config',
+    line    => 'Include /etc/ssh/sshd_config.d/*.conf',
+    notify  => Service['sshd'],
+    require => File['/etc/ssh/sshd_config.d/'],
+  }
+  file { '/etc/ssh/sshd_config.d/':
+    ensure => directory,
+    mode   => '0755',
```

Comment:
The mode for that directory by default is `0700`.

Reply:
fixed